lnwire/your_peer_storage.go

```diff
@@ -0,0 +1,63 @@
+package lnwire
+
+import (
+	"bytes"
+	"encoding/binary"
+	"io"
+)
+
+// YourPeerStorageBlob is the type of the data stored by peers as backup for
+// other peers. This message is sent in response to the PeerStorage message.
+type YourPeerStorageBlob []byte
```

Comment:
Yeah I especially think we should avoid having two *distinct* newtypes for this since this should be *identical* in both places. Namely, the contents of YourPeerStorageBlob should *exactly match* the contents of PeerStorageBlob in a prior message.